src/dune_file_watcher/dune_file_watcher.ml

```diff
@@ -358,6 +358,7 @@ let fswatch_backend () =
   | None ->
     let hints =
       match Platform.OS.value with
+      | Haiku -> [ Pp.text "fswatch is available on HaikuPorts" ]
```

Comment:
Binary packages are available, too. `pkgman install devel:libfswatch` will install the development files, or `pkgman install cmd:fswatch` for just the command.

Reply:
Thanks for taking a look! I'll try seeing which one is needed and update the message here accordingly.